src/tic_tac_toe/input/console.clj

```diff
@@ -34,11 +29,20 @@
     (if (valid-selection input board)
       (matrix-convrt input (board-diemensions board))
       (do (print-message "invalid selection")
-      	   (user-input-move board)))))
+      	  (user-input-move board)))))
+
+(defn game-key-to-strings [games]
+  (apply str (map #(str (inc (.indexOf games %)) "." (name %) "\n") games)))
+
+(defn get-game-selection [games]
+  (let [selection (read-string (prompt-terminal (str "select Game type\n" (game-key-to-strings games))))]
```

Comment:
This may be more clear if you either break it up into more lines in the let or create some extra private functions. Also, the capitalization in your message is kind of strange.